UpdatedSyntheticDataset/SyntheticDataset2/ElementsCreator/specified_letter_generator.py

```diff
@@ -0,0 +1,41 @@
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageFont
+from SyntheticDataset2.ImageOperations import *
+import random
+
+class SpecifiedLetterGenerator(object):
```

Comment:
This is mostly duplicated code from the ```random_letter_generator.py``` file. You should use an base class and extend it in this class and the above class to remove the code duplication.